root/static/scripts/common/i18n/admin.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * @flow strict
+ * Copyright (C) 2023 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import cleanMsgid from './cleanMsgid.js';
+
+export const l_admin: (string) => string =
+  cleanMsgid;
+
+export {l_admin as l};
+
+export function ln_admin(skey: string, pkey: string, val: number): string {
+  return val > 1 ? cleanMsgid(pkey) : cleanMsgid(skey);
+}
+
+export {ln_admin as ln};
+
+// eslint-disable-next-line no-unused-vars -- map context
+export const lp_admin = (key: string, context: string): string => (
+  cleanMsgid(key)
+);
+
+export {lp_admin as lp};
+
+export const N_l_admin = (key: string): (() => string) => (
+  () => l_admin(key)
```

Comment:
Something I'm not sure I get is the point of having both `l_admin` and `N_l_admin` (and likewise for ln, lp), if both are functionally no-ops and the strings aren't meant to be extracted. I feel that it might make sense to rename `l_admin` to `N_l_admin` (and remove the current `N_l_admin`) to make it clear that these functions are always no-ops. (Returning a function isn't needed for the same reasons.)

Reply:
`l_admin` isn’t exactly no-op though, it is `cleanMsgid`.

Reply:
That's right. I was thinking more in terms of no-localization. :)

I'm not that concerned about which name to use, anyway, though it still seems like we don't need both of these.

Reply:
Okay, I dropped all the `N_*_admin`.